src/test/osd/osd-scrub-repair.sh

```diff
@@ -2228,7 +2228,20 @@ function TEST_periodic_scrub_replicated() {
 
     # Request a regular scrub and it will be done
     pg_scrub $pg
-    grep -q "Regular scrub request, losing deep-scrub details" $dir/osd.${primary}.log || return 1
+    # the probability of flipping tail (not being scheduled) for 10 times in a
+    # raw is (2/3)^60 = 2.7e-11, which is approximately 0.
+    local -a delays=($(get_timeout_delays 60 1))
```

Comment:
In the spirit of keeping "make check" as fast as possible it seems like starting with 60 second sleep is too much.